zeppelin-plugins/launcher/docker/src/main/java/org/apache/zeppelin/interpreter/launcher/DockerInterpreterProcess.java

```diff
@@ -190,7 +192,7 @@ public void start(String userName) throws IOException {
     // check if the interpreter process exit script
     // if interpreter process exit, then container need exit
     StringBuilder sbStartCmd = new StringBuilder();
-    sbStartCmd.append("sleep 10; ");
+    sbStartCmd.append("sleep 20; ");
```

Comment:
By the way, I always wonder if we could have a better way to wait until the process is ready, not just praying :-)

Reply:
For Docker mode, there might be a way to actively execute the process instead of just waiting. 
However, in this case, we may have chosen this structure to upload interpreter files by using commands like `rm`, `mkdir`, while the container is running, and then execute the `interpreter.sh`. 

I will consider whether we can change this structure.